cmdstanpy/utils.py

```diff
@@ -672,7 +672,8 @@ def parse_method_vars(names: Tuple[str]) -> Dict[str, Tuple[int]]:
     return {v: tuple([k]) for (k, v) in enumerate(names) if v.endswith('__')}
 
 
-def parse_stan_vars(names: Tuple[str]) -> (Dict[str, Tuple[int]], Dict[str, Tuple[int]]):
+def parse_stan_vars(names: Tuple[str]) ->
+    (Dict[str, Tuple[int]], Dict[str, Tuple[int]]):
```

Comment:
I believe this should be 

`Tuple[Dict[str, Tuple[int,...], Dict[str, Tuple[int,...]` 

`Tuple[int]` is the type of a tuple with exactly one integer element, the ... makes it variable length of all ints. Ugly, I know. 

It may be worth adding something like mypy to the CI just to catch our own declarations

Reply:
mypy found 139 errors.  will try not to introduce new ones in this PR.

Reply:
I can go through and address existing ones after this PR